test/connectivity/scripts/connectivityTest.sh

```diff
@@ -52,6 +52,14 @@ function prepare_test_from_artifacts() {
     sed -i -e "s@<LogAnalyticsSharedKey>@$LOG_ANALYTICS_SHAREDKEY@g" "$deployment_working_file"
     sed -i -e "s@<UpstreamProtocol>@$UPSTREAM_PROTOCOL@g" "$deployment_working_file"
     
+    if [[ ! -z "$CUSTOM_EDGE_AGENT_IMAGE" ]]; then
+        sed -i -e "s@\"image\":.*azureiotedge-agent:.*\"@\"image\": \"$CUSTOM_EDGE_AGENT_IMAGE\"@g" "$deployment_working_file"
```

Comment:
Assuming this regex won't continue grabbing past the newline?

Reply:
yes.